modules/core/arrow-test/src/main/kotlin/arrow/test/UnitSpec.kt

```diff
@@ -1,9 +1,11 @@
 package arrow.test
 
+import arrow.data.NonEmptyList
 import arrow.typeclasses.Eq
 import arrow.test.laws.Law
 import io.kotlintest.TestCase
 import io.kotlintest.specs.StringSpec
+import java.util.*
```

Comment:
Is this necessary?

Reply:
We use `UUID` but we can remove the wildcard import for a concrete one.

Reply:
Oh this is in the tests. Nevermind then.